client/stylesheets/_toCat.scss

```diff
@@ -299,10 +299,9 @@ code {
 }
 
 .profileBtn {
-    background-color: $color-venetian-red !important;
-    border-color: $color-venetian-red !important;
+    background-color: #f8504b !important;
```

Comment:
Color literals like `#f8504b` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces